test/b2gedmntuples_cfg.py

```diff
@@ -275,18 +258,29 @@
 ### ------------------------------------------------------------------
 from JMEAnalysis.JetToolbox.jetToolbox_cff import jetToolbox
 
+listBtagDiscriminators = [ 'pfTrackCountingHighEffBJetTags',
+		'pfTrackCountingHighPurBJetTags',
+		'pfJetProbabilityBJetTags',
+		'pfJetBProbabilityBJetTags',
+		'pfSimpleSecondaryVertexHighEffBJetTags',
+		'pfSimpleSecondaryVertexHighPurBJetTags',
+		'pfCombinedSecondaryVertexV2BJetTags',
+		'pfCombinedInclusiveSecondaryVertexV2BJetTags',
+		'pfBoostedDoubleSecondaryVertexAK8BJetTags'
+		]
+
```

Comment:
This is in principle fine, but is there a time issue running the jet tool box? When I ran last I found it way slower than the older b2gedmntuples workflow. If it saves time, can consider dropping not  used b-taggers